### ISSUE TITLE
Error in docs code example for Uncontrolled editor

### ISSUE THREAD

**Issue by PrimeLens** [jpuri/react-draft-wysiwyg#443]
Hi - I'm still trying to figure out how to edit an already saved draft post when I came across an error in the examples in the docs. Here are the steps to recreate the issue and I've included a [link](https://github.com/PrimeLens/error-in-draft-wysiwyg-uncontrolled-editor) to a repo that demonstrates the error

Steps to recreate 

1) create-react-app draft-wysiwyg-error-uncontrolled-editor
2) cd draft-wysiwyg-error-uncontrolled-editor
3) npm install --save react-draft-wysiwyg
4) npm install --save draft-js
5) go to `https://jpuri.github.io/react-draft-wysiwyg/#/docs` and copy paste the `Uncontrolled editor using RawDraftContentState.` code block example into App.js
6) change `class UncontrolledEditor` to `class App`
7) npm run start

<img width="1169" alt="screengrab_of_error" src="https://user-images.githubusercontent.com/636249/30077659-890421c2-924a-11e7-95b6-c60159a17470.png">

**Comment by jpuri** [comment]
Hey @PrimeLens , do you also see this error in the docs page ?

**Comment by PrimeLens** [comment]
yes, this is copy paste from the docs

**Comment by jpuri** [comment]
Since this error does not happens in docs, looks like trouble in way its copied.

**Comment by PrimeLens** [comment]
Yes that is true ... but when a user _copies and pastes_ your code from the docs page into a brand new react app the error occurs.  

I have made simple steps to replicate.

I provided a simple repo to demonstrate.

**Comment by PrimeLens** [comment]
Sorry I edited my post above.

When you wrote "this error does not happens in docs" it is incorrect it _is_ happening in the docs. Possibly you were thinking this does not happen in the demo's?  The demo's do not cover this code from your docs. The only place it exists that I can see is by going to `Uncontrolled editor using RawDraftContentState` at `https://jpuri.github.io/react-draft-wysiwyg/#/docs` and copy pasting.

**Comment by jpuri** [comment]
Plz check example - 2 [here](https://jpuri.github.io/react-draft-wysiwyg/#/demo)

**Comment by PrimeLens** [comment]
Example 2 uses convertFromRaw,  your docs I pointed out use  RawDraftContentState
Please read my comments carefully

**Comment by divinzer** [comment]
same
